deck_chores/main.py

```diff
@@ -288,6 +288,14 @@ def main() -> None:  # pragma: nocover
         log_handler.setFormatter(logging.Formatter(cfg.logformat, style='{'))
         log.debug(f'Config: {cfg.__dict__}')
 
+        if cfg.stderr_lvl != 'NOTSET':
+            level = logging.getLevelName(cfg.stderr_lvl)
+            log_err = logging.StreamHandler(sys.stderr)
+            log_err.setLevel(level)
+            log_err.setFormatter(logging.Formatter(cfg.logformat, style='{'))
+            log.addHandler(log_err)
+            log_handler.addFilter(ExcludeErrorsFilter())
+
```

Comment:
1) wouldn't it be possible to setup that logger unconditionally? `NOTSET` should be a valid value as log level.
2) it seems reasonable to me, to move this setup into a function in the `utils` module and call it from here.
3) two equally configured formatters are used now.